### PR TITLE
[PL] translation of functions lesson

### PR DESCRIPTION
**Pull request by Koziolek**
In scope of issue #202

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @Koziolek!  cc @kdebowski for review

File: pl/lessons/basics/functions.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
+---
+layout: page
+title: Funkcje
+category: basics
+order: 7
+lang: pl
+---
+
+W Elixirze tak, jak w wielu innych językach funkcyjnych, funkcje należą do bytów podstawowych (ang. _first class
```
Comment (kdebowski) [inline pl/lessons/basics/functions.md]: Comma is in bad place. It's:
W Elixirze tak, jak w...
Should be:
W Elixirze, tak jak w...

File: pl/lessons/basics/functions.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
+---
+layout: page
+title: Funkcje
+category: basics
+order: 7
+lang: pl
+---
+
+W Elixirze tak, jak w wielu innych językach funkcyjnych, funkcje należą do bytów podstawowych (ang. _first class 
+citizen_). W tej lekcji poznamy rodzaje funkcji, różnice pomiędzy nimi oraz zastosowania.
```
Comment (kdebowski) [inline pl/lessons/basics/functions.md]: Is it a new line or diff is displayed in such way? This should be continuation of previous line.

File: pl/lessons/basics/functions.md (hunk 1 above)
Comment (Koziolek) [inline pl/lessons/basics/functions.md]: just display plus line wrapping in editor. It has no influence to final look&feel of text.

**Comment by Koziolek** [comment]
Done. Merged

**Pull request merged**